site/jekyll/guides/server-side-rendering.md

```diff
@@ -72,3 +72,26 @@ meaning your initial render will be super fast.
 
 For a more in-depth example, take a look at the included sample application
 (**React.Samples.Mvc4**).
+
+5 - How not to render the React data attributes
+
+If thre is no need to have a React application client side and you just want to use the server side rendering but without the React specific data attributes call `Html.React` and pass renderReactAttributes parameter as false.
+
+```csharp
+@Html.React("HelloWorld", new
+{
+	name = "Daniel",
+ 	renderReactAttributes: false
```

Comment:
I don't think this example works as-is, you're defining `renderReactAttributes` inside the props rather than as a separate parameter.